File: occupancy-manager-java/src/main/java/com/occupancy/manager/service/IOccupancyManagerService.java
package com.occupancy.manager.service;

import com.occupancy.manager.dto.OccupancyUsageRequest;
import com.occupancy.manager.dto.OccupancyUsageResponse;

/**
 * 
 * @author Ramachandra
 *
 */
public interface IOccupancyManagerService {

	OccupancyUsageResponse calculateOccupancy(OccupancyUsageRequest occupancyUsageRequest);
}

File: occupancy-manager-kotlin/src/main/kotlin/com/occupancy/manager/service/IOccupancyManagerService.kt
package com.occupancy.manager.service

import com.occupancy.manager.dto.OccupancyUsageRequest
import com.occupancy.manager.dto.OccupancyUsageResponse

/**
 *
 * @author Ramachandra
 */
interface IOccupancyManagerService {
	fun calculateOccupancy(occupancyUsageRequest: OccupancyUsageRequest): OccupancyUsageResponse
}